### PR TITLE
Ignore godeps directory

### PR DESCRIPTION
**Pull request by jeffreymelvin-wf**
@stevenosborne-wf 
@infosec-wf @matthewlorimor-wf

### CODE REVIEW

**Comment by aviary2-wf** [comment]
## Raven
<details>
<summary>Number of Findings: 1</summary>
<li>Watched file aviary.yaml modified</li>
</details>

**Comment by brianshannan-wf** [comment]
This needs to be made against the develop branch.

**Comment by matthewlorimor-wf** [comment]
@jeffreymelvin-wf ^^^

File: aviary.yaml [hunk 1]
```diff
@@ -2,6 +2,9 @@ version: 1
 
 exclude:
   - tests?/
+  - Godeps/
+  - ^examples/
+  - ^vendor/
```
Comment (brianshannan-wf) [inline aviary.yaml]: Can this just be `vendor/` in case we add a vendor directory for subdirectories in the future like `lib/go/` or `example/go/`?

**Pull request closed without merging**